ush/setup.sh

```diff
@@ -1496,7 +1496,8 @@ NDIGITS_ENSMEM_NAMES="0"
 ENSMEM_NAMES=("")
 FV3_NML_ENSMEM_FPS=("")
 if [ "${DO_ENSEMBLE}" = "TRUE" ]; then
-  NDIGITS_ENSMEM_NAMES="${#NUM_ENS_MEMBERS}"
+#  NDIGITS_ENSMEM_NAMES="${#NUM_ENS_MEMBERS}"
+  NDIGITS_ENSMEM_NAMES="2"
```

Comment:
so the latest plan is to hardwire NDIGITS_ENSMEM_NAMES to 2?

Reply:
Reversed back to the old way.